static/src/javascripts/projects/common/modules/experiments/tests/recommended-for-you.js

```diff
@@ -112,45 +140,67 @@ define([
             );
         }
 
-        function imageUrlFromItem(item) {
-            function imageFromTemplate(img) {
-                return img.replace('#{width}', 220).replace('#{height}', 146).replace('#{quality}', 0.8);
-            }
-            if (item.headerImage) {
-                return imageFromTemplate(item.headerImage.urlTemplate);
-            } else if (item.headerVideo) {
-                return imageFromTemplate(item.headerVideo.stillImage.urlTemplate);
-            } else {
-                return null;
-            }
-        }
-
-        function itemFromRecommendationItem(item) {
-            return {
-                'id': item.item.id,
-                'imageUrl': imageUrlFromItem(item.item),
-                'title': item.item.title,
-                'standFirst': item.item.standFirst
-            };
-        }
-
         function setupComponentAttentionTracking(trackingCode) {
             require(['ophan/ng'], function (ophan) {
                 ophan.trackComponentAttention(trackingCode, $recommendedForYouSection[0]);
             });
         }
 
         function insertSection(items) {
+            var $oldSection = $recommendedForYouSection;
+
             $recommendedForYouSection = $.create(template(recommendedForYouTemplate, {
```

Comment:
What are the benefits of mutating `$recommendedForYouSection`, rather than defining a local var?

e.g.

```
function insertSection(items) {
    var localVarForRecommendSection = createRecommendedForYouContent(items);

    if ($recommendedForYouSection) {
        return fastdom.write(function() {
            $recommendedForYouSection.replaceWith(localVarForRecommendSection);
            setupComponentAttentionTracking('recommended-for-you_user-history');
        });
    } else {
        return fastdom.write(function() {
            localVarForRecommendSection.insertBefore($opinionSection);
            setupComponentAttentionTracking('recommended-for-you_user-history');
        });
    }
}
```